src/pages/MyAccount/MyAccount.jsx

```diff
@@ -1,14 +1,57 @@
 import React, { useEffect, useState } from 'react';
 import My from '../../components/My/My.jsx';
-import { APIS } from '../../config.js';
 import { M } from '../../components/My/My';
 import { S } from './MyAccount';
+import { LuImagePlus } from 'react-icons/lu';
+import { APIS } from '../../config';
 
 const MyAccount = () => {
-  const [userInfo, setUserInfo] = useState({});
+  const [userInfo, setUserInfo] = useState([]);
+  const [isModify, setIsModify] = useState(false);
+  const [userName, setUserName] = useState('');
+  const [imgFile, setImgFile] = useState({});
+
+  const handleFileChange = e => {
+    const profileImg = e.target.files[0];
+    if (!userInfo || userInfo.length === 0) {
+      return;
+    }
+    setImgFile(profileImg);
+  };
+
+  const handleNameChange = e => {
+    setUserName(e.target.value);
+  };
+
+  const handleModify = e => {
+    const formData = new FormData();
+
+    const NameData = { nickname: userName };
+    const data = JSON.stringify(NameData);
+
+    formData.append('profileImageUrl', imgFile);
+    formData.append('data', data);
+
+    if (userName !== '') {
+      fetch('http://10.58.52.122:3000/users', {
+        method: 'PATCH',
+        headers: {
+          Authorization: localStorage.getItem('accessToken'),
+        },
+        body: formData,
+      })
+        .then(res => res.json())
+        .then(data => {
+          if (data.message === 'Update UserInfo') {
+            alert('정보가 수정되었습니다!');
+            setIsModify(false);
+          }
+        });
+    } else return;
```

Comment:
```suggestion
    }
```
조건이 아닐때 어떤 것을 하지 않으면 이렇게 작성하셔도 됩니다